flexget/components/sites/sites/hebits.py

```diff
@@ -27,220 +17,151 @@
 
 
 @unique
-class HEBitsCategory(Enum):
-    pc_games = 21
-    console_games = 33
-    hd_movies = 27
-    bluray = 36
-    dvdr = 20
-    sd_movies = 19
-    il_movies = 25
-    top_imdb = 34
-    hd_shows = 1
-    hd_shows_subs = 24
-    il_hd_shows = 37
-    il_shows = 7
-    animated = 23
-    music = 28
-    flac_music = 31
-    il_music = 6
-    concerts = 35
-    soundtrack = 30
-    mobile = 32
-    ebooks = 26
-    apps = 22
-    misc = 29
-    xxx = 9
-    sport = 41
-
-
-class HEBitsSort(Enum):
-    names = 1
-    files = 2
-    comments = 3
-    date = 4
-    size = 5
-    completed = 6
-    uploaders = 7
-    downloaders = 8
-    uploader = 9
-
-
-class HEBitsCookies(Base):
-    __tablename__ = 'hebits_cookies'
-
-    user_name: str = Column(Unicode, primary_key=True)
-    _cookies = Column('cookies', Unicode)
-    cookies: dict = json_synonym('_cookies')
-    expires: datetime = Column(DateTime)
+class HeBitsCategory(Enum):
+    movies = 1
+    tv = 2
+    theater = 3
+    software = 4
+    games = 5
+    music = 6
+    books = 7
+    movies_packs = 8
+    porno = 9
+    other = 10
+
+
+class HeBitsSort(Enum):
+    time = 1
+    year = 2
+    size = 3
+    snatched = 4
+    seeders = 5
+    leechers = 6
+    random = 7
 
 
 class SearchHeBits:
-    """
-    HEBits search plugin.
-    """
+    """HeBits search plugin.
+
+    ```yaml
```

Comment:
These are only visible from commandline help. You can't use formatting and should keep it short (no complete examples with other plugins).